scout/constants/acmg.py

```diff
@@ -309,48 +309,43 @@
 
 ACMG_POTENTIAL_CONFLICTS = [
     (
-        "PVS1",
-        "PM4",
+        {"PVS1", "PM4"},
```

Comment:
refactored this to a list, so you can have as many conflicts as you want instead of just 2